c2corg_ui/templates/account.html

```diff
@@ -0,0 +1,40 @@
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+<%block name="pagetitle"><title ng-bind="mainCtrl.page_title('Account')">${show_title('Account')}</title></%block>
+<br /><br /><br /><br />
```

Comment:
Right, the appearance would need some love.
The final design of the page is not yet decided so I just did the minimum to avoid the floating bar at the top.
I will add some temporary CSS then.